### PR TITLE
feat(tutorial): complete step 3

### PR DESCRIPTION
**Pull request by heinsj**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/RepoPage/RepoPage.js [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+          createdAt
+          description
+          id
+        }
+      }
+    }
+  }
+`;
+
+const LinkList = ({ url, homepageURL }) => (
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: The case of the var `homepageUrl` does not match the definition in the REPO_QUERY query.

File: src/content/RepoPage/RepoPage.js [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+);
+
+const getRowItems = rows =>
+  rows.map(row => ({
+    ...row,
+    key: row.id,
+    stars: row.stargazers.totalCount,
+    issueCount: row.issues.totalCount,
+    createdAt: new Date(row.createdAt).toLocaleDateString(),
+    updatedAt: new Date(row.updatedAt).toLocalDateString(),
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: typo in function name - should be `toLocaleDateString()`

**Comment by heinsj** [comment]
@kellychurchill Thank you for finding these issues. I knew it would be a typo or 2 somewhere but I just couldn't see them. I am still getting the same error in my local environment but it looks correct when I preview above. 

Do you have any suggestions for my local environment?

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.  

I don't know why you might still be seeing errors locally. Perhaps remove your node_modules folder and then do `yarn` again?

**Comment by heinsj** [comment]
I figured out the main auth problem. I was missing an “=” between the key and my personal access token. I am all set now.

**Pull request closed without merging**